BLSQ/iaso
# [IA-3782] Bulk push of GPS from submissions to OUs - backend

Added a new instance endpoint for bulk pushing GPS data to orgunits

Related JIRA tickets : IA-3782, IA-3667 (parent ticket)

## Self proofreading checklist


## Doc

Some comments and API sketches are in the parent ticket.

## Changes

Added a new instance endpoint: `/api/instances/check_bulk_gps_push/`
Added a new task endpoint: `api/tasks/create/instancebulkgpspush/`

## How to test

Go to the web API, try to use the endpoint with available filters (`select_all`, `selected_ids` and `unselected_ids`).
Make sure that you have location and orgunits attached to your instances.
See unit tests to see various use cases.

After running the task (with an active worker!), make sure that the changes you requested were applied on the org units.


## Print screen / video
![image](https://github.com/user-attachments/assets/5d066b64-514a-496e-b776-4ff464126ed5)


## Notes
/


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
